crypto/keys/sr25519/encoding.go

```diff
@@ -7,6 +7,9 @@ import (
 var _ crypto.PrivKey = PrivKey{}
 
 const (
+	PrivKeyName = "cosmos-sdk/PrivKeySr25519"
+	PubKeyName  = "cosmos-sdk/PubKeySr25519"
```

Comment:
@marbar3778 you mention in https://github.com/cosmos/cosmos-sdk/issues/6886#issuecomment-671616446 that you'd like to keep maybe offer SR25519 as validator key. Is there a risk that these keys go back to tendermint one day?

Reply:
The key name is mainly for amino and Tendermint doesn't use amino anymore. It should be fine, once we support it we will use our impl.